patches/src/rdkafka_broker.c.patch

```diff
@@ -1,21 +1,21 @@
 diff --git a/src/rdkafka_broker.c b/src/rdkafka_broker.c
-index 481c21d9..765b1690 100644
+index 481c21d9..01bb7b49 100644
 --- a/src/rdkafka_broker.c
 +++ b/src/rdkafka_broker.c
 @@ -38,7 +38,7 @@
   * src/rd.h defines _POSIX_SOURCE to be 200809L, which corresponds to XPG7,
   * which itself is not compatible with _XOPEN_SOURCE on that platform.
   */
 -#if !defined(_AIX) && !defined(__sun)
-+#if !defined(_AIX) && !defined(__sun) && !defined(_ZOS)
++#if !defined(_AIX) && !defined(__sun) && !defined(__MVS__)
  #define _XOPEN_SOURCE
  #endif
  #include <signal.h>
 @@ -49,6 +49,10 @@
  #include <string.h>
  #include <ctype.h>
  
-+#ifdef _ZOS
++#ifdef __MVS__
 +#include <strings.h>
```

Comment:
What is strings.h needed for?

Reply:
If I don't include, I get a warning for strcasecmp()

Reply:
I have pushed the changes, can you please review